stats/test/platform-routes.test.js

```diff
@@ -153,6 +157,50 @@ describe('Platform Routes HTTP request handler', () => {
     })
   })
 
+  describe('GET /participants/top-measurements', () => {
+    it('returns top measurement stations for the given date', async () => {
+      const today = new Date()
+      const yesterday = new Date()
+      yesterday.setDate(today.getDate() - 1)
+
+      await givenDailyStationMetrics(pgPools.evaluate, yesterday.toISOString().split('T')[0], [
+        { ...STATION_STATS, stationId: 's3', participantAddress: 'f1ghijkl', acceptedMeasurementCount: 50 },
+        { ...STATION_STATS, acceptedMeasurementCount: 20 },
+        { ...STATION_STATS, stationId: 's2', acceptedMeasurementCount: 30 },
+        { ...STATION_STATS, stationId: 's2', inetGroup: 'group2', acceptedMeasurementCount: 40 }
+      ])
+      await givenDailyStationMetrics(pgPools.evaluate, today.toISOString().split('T')[0], [
+        { ...STATION_STATS, acceptedMeasurementCount: 10 }
+      ])
+
+      await pgPools.evaluate.query('REFRESH MATERIALIZED VIEW top_measurement_participants_yesterday_mv')
+
+      // We don't care about the date range for this query
+      const res = await fetch(
+        new URL(
+          '/participants/top-measurements?from=2024-01-11&to=2024-01-11',
```

Comment:
> We don't care about the date range for this query

As I am arguing in my other comment above, we need to care about the date range.